### PR TITLE
Add EncoderCount.msg

### PR DESCRIPTION
**Pull request by markwomack**

### CODE REVIEW

File: victoria_sensor_msgs/msg/EncoderCount.msg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This message represents the count value
+# for an encoder.
+
+std_msgs/Header header
+int64 count
```
Comment (griswaldbrooks) [inline victoria_sensor_msgs/msg/EncoderCount.msg:5]: Ha, sorry, can you add a max?

**Review by griswaldbrooks — CHANGES_REQUESTED** [review]

File: victoria_sensor_msgs/msg/EncoderCount.msg (hunk 1 above)
Comment (markwomack) [reply victoria_sensor_msgs/msg/EncoderCount.msg:5]: No min?

Comment (griswaldbrooks) [reply victoria_sensor_msgs/msg/EncoderCount.msg:5]: I would have suggested min but you're use of an `int64` says the min is zero, unless the encoder would reset higher than that, which... I don't **think** makes any sense.

Comment (markwomack) [reply victoria_sensor_msgs/msg/EncoderCount.msg:5]: Actually, int64 can go negative.  I think I should add a min.

Comment (griswaldbrooks) [reply victoria_sensor_msgs/msg/EncoderCount.msg:5]: Strange, I don't feel drunk. Add the min.

Comment (markwomack) [reply victoria_sensor_msgs/msg/EncoderCount.msg:5]: Done.

**Pull request merged**